Capybara_App.service

```diff
@@ -0,0 +1,16 @@
+#Type:idle - systemd will delay execution of the service binary until all jobs are dispatched
+#Restart Capybara_App service always if the program is on-failure or terminated
+#multi-user.target - running on runlevel-3, start after the system is boot-up
+
+[Unit]
+Description=Embedded Capybara App to Systemd service
+After=multi-user.target
+
+[Service]
+Type=idle
+ExecStart=/usr/bin/python3 /home/pi/newClone/capybara/app/main.py
```

Comment:
So this line is super important. specifically this `/home/pi/newClone/capybara/app/main.py` we need to make sure in the setup of the raspberry pies that we always put the files in that location.  I would maybe sugges that we install the file to a common directory like `/var/` instead of the users home directory but it's not that important.